src/topology/metric_space/basic.lean

```diff
@@ -435,6 +435,45 @@ begin
   exact ⟨_, ⟨F ⟨x, xs⟩, rfl⟩, hF _ _ this.symm⟩
 end
 
+/-- Defining locally uniform convergence on a set using `dist`. -/
```

Comment:
Do you mean "Expressing" here?

Reply:
Yes, here and below and in the other file. Changed.